pcstac/tests/data-files/naip/items/al_m_3008501_ne_16_060_20191109_20200114_relative_path_test.json

```diff
@@ -0,0 +1,106 @@
+{
+    "type": "Feature",
+    "stac_version": "1.0.0",
+    "id": "al_m_3008501_ne_16_060_20191109_20200114_relative_path_test",
+    "properties": {
+      "naip:state": "al",
+      "providers": [
+        {
+          "name": "USDA Farm Service Agency",
+          "roles": [
+            "producer",
+            "licensor"
+          ],
+          "url": "https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/"
+        }
+      ],
+      "proj:epsg": 32616,
+      "datetime": "2019-11-09T00:00:00Z"
+    },
+    "geometry": {
+      "type": "Polygon",
+      "coordinates": [
+        [
+          [
+            -85.875,
+            30.9375
+          ],
+          [
+            -85.875,
+            31.0
+          ],
+          [
+            -85.9375,
+            31.0
+          ],
+          [
+            -85.9375,
+            30.9375
+          ],
+          [
+            -85.875,
+            30.9375
+          ]
+        ]
+      ]
+    },
+    "links": [
```

Comment:
Just for clarity, a good test would be to add a new link here with a relative path to another item (`collections/naip/items/{item_id}`).

Reply:
It doesn't look like my original main review comment made it into the review - sorry if this test comment was confusing. I've re-added a comment about asset vs link hrefs in this follow-up review.